### ISSUE TITLE
v2.4.0/ windows Debugging mode can launch the page normally, but after packaging using publishing mode, open the application error.

### ISSUE THREAD

**Issue by conero** [wailsapp/wails#2505]
### Description

Debugging mode can launch the page normally, but after packaging using publishing mode, open the application error. Like the picture: 

![image](https://user-images.githubusercontent.com/19564821/226183441-2dfe4f5f-79e8-433a-ad7b-a10d096fe6ef.png)

And I try to use `wails doctor` that unfound error mesaage.

### To Reproduce

1. Run `wails build` and done noting error.
2. Launch application prompt abnormality: 

```json
{"error":"Request","message":"找不到任何可成功连接的IP (wails.localhost:80)"}

```

### Expected behaviour

1. Start the application normally
2. When an exception occurs, it can debug the error log on the released version or provide more detailed error prompts

### System Details

```shell
DEB | Using go webview2loader
Wails CLI v2.4.0

Scanning system - Please wait (this may take a long time)...Done.

# System

OS           | Windows 10 Home China
Version      | 2009 (Build: 22000)
ID           | 21H2
Go Version   | go1.20
Platform     | windows
Architecture | amd64

# Wails

Version | v2.4.0

# Dependencies

Dependency | Package Name | Status    | Version
WebView2   | N/A          | Installed | 111.0.1661.44
npm        | N/A          | Installed | 8.18.0
*upx       | N/A          | Available |
*nsis      | N/A          | Available |
* - Optional Dependency

# Diagnosis

Your system is ready for Wails development!
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
  - nsis : More info at https://wails.io/docs/guides/windows-installer/
```

**Comment by caobingsheng** [comment]
me too

**Comment by styfstr** [comment]
me too.
![image](https://user-images.githubusercontent.com/44523785/226232183-5e329ba3-71b9-47ae-a262-56366d605103.png)

**Comment by stffabi** [comment]
Thanks for using Wails and reporting this issue.

The latest Webview2 Runtime release (>110) broke some part of the assetserver and we merged a fix for it with PR #2496. I suspect this might be the issue.
Would it be possible for you to test the master commit, with the the following [guide](https://wails.io/docs/guides/local-development)?

**Comment by leaanthony** [comment]
I think we should release this fix as soon as we can 😄

**Comment by leaanthony** [comment]
This is now fixed in v4.2.1 👍
